.gitignore

```diff
@@ -6,3 +6,6 @@ npm-debug.log
 .tern-project
 yarn-error.log
 .vscode/
+manifest.yml
+.imdone/
+config.local.yml
```

Comment:
`config.local.yml` is storing my backend config so I don't have to go searching for it to test with GitHub.